sympy/logic/inference.py

```diff
@@ -77,11 +77,12 @@ def satisfiable(expr, algorithm=None, all_models=False, minimal=False):
     UNSAT
 
     """
+def satisfiable(expr, algorithm=None, all_models=False, minimal=False):
     if algorithm is None or algorithm == "pycosat":
-        pycosat = import_module('pycosat')
-        if pycosat is not None:
+        try:
```

Comment:
Also, I don't really see how this improves the code in general.  Using a lone `try-except` instead of `if-else` for a logical condition makes the code harder to read and increases the time report.
Maybe other members can tell if this is an apt refactoring.

Reply:
The `import_module` function doesn't raise `ImportError` so it isn't even possible to catch the exception from it.